be/src/vec/functions/function_encryption.cpp

```diff
@@ -335,7 +335,7 @@ struct EncryptionAndDecryptFourImpl {
         return Status::OK();
     }
 
-    static void vector_const(const ColumnString* column, StringRef iv_arg, StringRef key_arg,
```

Comment:
we should add: `use_default_implementation_for_constants to false`?

otherwise we still can not go to vector_const branch.

Reply:
we can. this is vector_const, not const_const

Reply:
please add some log or have a debug to make sure it.

the default const impl will convert const columns to normal columns at first, then call execute_impl function.

<img width="1271" alt="image" src="https://github.com/user-attachments/assets/9cc02f26-5b36-4ca8-8a5c-ee291a376fe5">